docs/source/install/k8s_ha.rst

```diff
@@ -0,0 +1,369 @@
+Please refer to `K8s <https://kubernetes.io/docs/home/>`__ and `Helm <https://docs.helm.sh/>`__
+documentation if you find any difficulties using these tools.
+
+However, here are some minimal instructions to get started.
+
+Deployment
+__________
+StackStorm HA cluster available as a Helm chart, a bundled K8s package which
+makes installing complex StackStorm infrastructure easy as:
+
+.. code-block:: bash
+
+  # Add Helm StackStorm repository
+  helm repo add stackstorm https://helm.stackstorm.com/
+
+  helm install stackstorm/stackstorm-ha
+
+Once the deployment is finished, it'll show you first steps how to start working with the new cluster via WebUI or st2 client:
+
+.. figure :: /_static/images/helm-chart-notes.png
+    :align: center
+
+
+The installation uses some unsafe defaults which we recommend you change thoughtfully for production use via Helm ``values.yaml``.
+
+Helm Values
+___________
+Helm package ``stackstorm-ha`` comes with default settings (see `values.yaml <https://github.com/StackStorm/stackstorm-ha/blob/master/values.yaml>`_).
+Fine-tune them to achieve desired configuration for the StackStorm HA K8s cluster.
+
+.. note::
+    Keep custom values you want to override in a separated yaml file so they won't get lost.
+    Example: ``helm install -f custom_values.yaml`` or ``helm upgrade -f custom_values.yaml``
+
+You can configure:
+
+- number of replicas for each component
+- st2 auth secrets
+- st2.conf settings
+- RBAC roles, assignments and mappings (enterprise only)
+- custom st2 packs and its configs
+- st2web SSL certificate
+- SSH private key
+- K8s resources and settings to control pod/deployment placement
+- configuration for Mongo, RabbitMQ clusters
+- configuration for in-cluster Docker registry
+
+.. warning::
+    It's highly recommended to set your own secrets as file contains unsafe defaults like self-signed SSL certificates, SSH keys, StackStorm access credentials and MongoDB/RabbitMQ passwords!
+
+Upgrading
+_________
+Once you make any changes to Helm values, upgrade the cluster:
+
+.. code-block:: bash
+
+  helm repo update
+  helm upgrade <release-name> stackstorm/stackstorm-ha
+
+It will redeploy components which were affected by the change, taking care to keep
+the desired number of replicas to sustain every service alive during the rolling upgrade.
+
+
+Enterprise (EWC)
+________________
+
+Installation
+~~~~~~~~~~~~
+By default, StackStorm Community free and open-source version is deployed via Helm chart.
+If you want to install :doc:`StackStorm Enterprise (Extreme Workflow Composer) </install/bwc>`, run:
+
+.. code-block:: bash
+
+  # Replace `<EWC_LICENSE_KEY>` with a real license key, obtained in Email
+  helm install \
+    --set enterprise.enabled=true \
+    --set enterprise.license=<EWC_LICENSE_KEY> \
+    stackstorm/stackstorm-ha
+
+It will pull enterprise images from private Docker registry as well as adds more advanced functionality and enterprise support.
```

Comment:
It will pull enterprise images from private Docker registry as well as add more advanced functionality and enterprise support.

☝️ replaced 'adds' with 'add'